ops/cloud-deployment/app-insights/app-insights.bicep

```diff
@@ -2,24 +2,138 @@ param location string = resourceGroup().location
 param appInsightsName string
 param workspaceResourceId string
 param applicationType string
-
+param kind string
 
 resource appInsights 'Microsoft.Insights/components@2020-02-02' = {
   name: appInsightsName
   location: location
-  kind: 'web'
+  kind: kind
   properties: {
     Application_Type: applicationType
-    DisableIpMasking: false
-    DisableLocalAuth: true
-    Flow_Type: 'Redfield'
-    ImmediatePurgeDataOn30Days: true
-    IngestionMode: 'LogAnalytics'
-    publicNetworkAccessForIngestion: 'Enabled'
-    publicNetworkAccessForQuery: 'Enabled'
-    Request_Source: 'IbizaAIExtensionEnablementBlade'
-    RetentionInDays: 90
-    SamplingPercentage: null
     WorkspaceResourceId: workspaceResourceId
+
   }
 }
+
+resource diagnosticLogs 'Microsoft.Insights/diagnosticSettings@2021-05-01-preview' = {
+  name: appInsights.name
+  scope: appInsights
+  properties: {
+    workspaceId: workspaceResourceId
+    logs: [
+      {
+        category: 'AppAvailabilityResults'
+        categoryGroup: null
+        enabled: true
+        retentionPolicy: {
+          days: 0
+          enabled: false
```

Comment:
Also, I would assume we'd want to have this enabled and removing logs after a given timeframe like 30 days